src/clarity-angular/popover/common/popover.directive.spec.ts

```diff
@@ -15,111 +16,45 @@ describe("Popover directive", () => {
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            declarations: [TestComponent, DesugaredSyntaxComponent, PopoverDirective]
+            declarations: [TestComponent, PopoverDirective],
+            providers: [IfOpenService]
         });
+
+        fixture = TestBed.createComponent(TestComponent);
+        fixture.detectChanges();
+        compiled = fixture.nativeElement;
     });
 
     afterEach(() => {
         fixture.destroy();
     });
 
     it("projects content", function () {
-        fixture = TestBed.createComponent(TestComponent);
-        compiled = fixture.nativeElement;
-
         expect(compiled.textContent).toMatch(/anchor1/);
-        expect(compiled.textContent).toMatch(/anchor2/);
-    });
-
-    it("shows popover content if open", function () {
-        fixture = TestBed.createComponent(TestComponent);
-        compiled = fixture.nativeElement;
-
-        fixture.componentInstance.open1 = true;
-        fixture.detectChanges();
-        expect(compiled.textContent).toMatch(/popover1/);
-        expect(compiled.textContent).not.toMatch(/popover2/);
     });
 
-    it("queues up opening of subsequent popovers if one is already open", function () {
-        fixture = TestBed.createComponent(TestComponent);
-        compiled = fixture.nativeElement;
-
-        fixture.componentInstance.open1 = true;
-        fixture.componentInstance.open2 = true;
-        fixture.componentInstance.open3 = true;
-        fixture.detectChanges();
+    it("positions popover only if the open property of ifOpenService is true", function () {
```

Comment:
All the actual positioning is done by `popover.ts` and is covered by `popover.spec.ts`.